### PR TITLE
version update and preliminary changelog for 3.0.0

### PR DESCRIPTION
**Pull request by Bob-Murphy**
### What have you changed and why?
- Changed version and added preliminary changelog for 3.0.0 release
- added 2.5.4 changelog which was prior only merged onto master
- added 2.5.5 changelog which was prior only merged onto master   

### What should the reviewers do?
- Check and complete the additional information of the Major changes
- Check for completion
- Check for typos
- Check for unnecessary information

### CODE REVIEW

**Comment by jaj22** [comment]
Provisional chunk for the attack/support rework. Skips most of the underlying changes, concentrating on stuff visible to players:
```
.. rubric:: Complete rework of the attack & support system
   * Enemy factions are now resource-limited:
      - Each faction has separate pools for attack and defence resources.
      - Any vehicle or unit spawned and/or destroyed has a cost related to their capability.
      - Resource income is dependent on war tier, aggro, active player count and difficulty.
      - Attack vs defence and occupant vs invader resource balances can be adjusted separately. 
      - Support system makes decisions based on available resources and location value.
      - Flag capture counterattacks are no longer automatic, instead depending on location and resources.
   * Vehicle selection improvements:
      - More gradual scaling of vehicle quality.
      - More ground and fewer air vehicles used, especially for factions with weak air options.
      - Proportions of transport and support vehicles are more controlled.
      - Punishments and HQ attacks may now include some ground vehicles.
   * Attack/support behaviour improvements:
      - Paratroopers (usually) drop further away and pull their chutes higher.
      - Transport helis land further from the target area.
      - Attack helis are less inclined to suicide into zu-23s.
      - Ground vehicle travel times reduced and infantry offload reliability improved.
      - CAS rewritten for reliability and fairness.
   * Attack target selection rewritten.
      - Enemies can now attack rebel targets outside mission distance at reduced probability.
   * Enemy HQ knowledge is now persistent:
      - Enemies may gain knowledge of HQ when supports are called nearby, or from traitor missions.
      - Once enemies are aware of the HQ, an HQ attack may be launched instead of a normal major attack.
      - Moving HQ more than 1km away will reset the HQ knowledge.
```

File: changelog.rst [hunk 1]
```diff
@@ -8,6 +8,300 @@ Changelog
 |
 
 
+Version 3.0.0
+===============
+
+.. note::
+   xxth Xxxxx 2022
+
+.. admonition:: Major
```
Comment (targetingsnake) [inline changelog.rst:17]: indents are missing for points under admonition, causes an error of sphinx in compilation, would add one indent to all rubrics till admonition 'Minor'

File: changelog.rst [hunk 2]
```diff
@@ -8,6 +8,300 @@ Changelog
...
+   * Czech
+   * French
+   * Italian
+   * Korean
+   * Polish
+   * Russian
+   * Simplified Chinese
+   * Spanish
+
+.. admonition:: Minor
```
Comment (targetingsnake) [inline changelog.rst:96]: missing empty line beneath

File: changelog.rst [hunk 3]
```diff
@@ -8,6 +8,300 @@ Changelog
...
+   * :issue:`2503` Added parameters for initial player and rebel faction  money
+   * :issue:`2510` Improved attack selection dependency on rebel HQ position
+   * :issue:`2521` Implemented AFK timeout parameter & status bar indicator to prevent AFK commanders blocking the progress
+   * :issue:`2523` KAT Medical implementation got updated to most current Kat - Advanced Medical REWRITE
+   * :issue:`2531` Adds logged in admins as members
+   * :issue:`2532` Balance utility trucks in cases where the civ factions lack them
+   * :issue:`2535` Increased default garage cap to 20 base + 4 per warlevel
+   * :issue:`2563` Added parameter for initial HR
+
+.. admonition:: Template updates
```
Comment (targetingsnake) [inline changelog.rst:132]: missing empty line beneath

File: changelog.rst [hunk 4]
```diff
@@ -8,6 +8,300 @@ Changelog
...
+      * Removed unused loadout creation stuff as its all handled by EquipRebell
+      * Removed comments as they can be found in Example Templates
+      * Updated format of the Rebel Example Template
+      * Added If cases for DLC uniforms for Vanilla and RHS
+      * Added a check in initVarServer for an empty civ helicopter as it will error with VN rebels
+      * Fixed a syntax error in the MACV template
+      * Fixed miscased classnames in Vanilla Ai templates
+      * Added faces and voices (speaker) for the AI in the templates
+
+.. admonition:: Map oupdates
```
Comment (targetingsnake) [inline changelog.rst:143]: missing empty line beneath

File: changelog.rst [hunk 5]
```diff
@@ -8,6 +8,300 @@ Changelog
...
+   * Tembelan Island
+      - moved markers
+   * Virolahti
+      - towns updated, added fuel stations, added bank locations, updated folder structure within sqm, updated vehicle markers, fixed broken marker names, removed edit-terrain-object-modules
+   * Chernarus_autumn
+      - added fresh map port
+   * Khe Sanh
+      - added fresh map port
+
+.. admonition:: Groundwork
```
Comment (targetingsnake) [inline changelog.rst:175]: missing empty line beneath

File: changelog.rst [hunk 6]
```diff
@@ -8,6 +8,300 @@ Changelog
...
+   * :issue:`2450` Changed spawning rules for airborne players and rebel UAVs
+   * :issue:`2459` Added game type definition
+   * :issue:`2460` Added safeguard to mod.cpp
+   * :issue:`2511` Switched over to config based logistic nodes
+   * :issue:`2534` Move the controls defined in setupDialog.hpp to control.hpp
+   * Removed a metric ton of old code
+   * Added assets and background functions for the UI rework.
+   * Set up a new build and publish pipeline on GitHub
+
+.. admonition:: Bugfixes
```
Comment (targetingsnake) [inline changelog.rst:201]: missing empty line beneath

File: changelog.rst [hunk 7]
```diff
@@ -8,6 +8,300 @@ Changelog
...
+   * :issue:`2498` Fixed players being able to mount a static that is being carried
+   * :issue:`2508` Fixed an infinite money exploit
+   * :issue:`2517` Fixed being able to load game saves with missing mods
+   * :issue:`2537` Fixed Antistasi UI layer numbers fighting with other mods
+   * :issue:`2551` Fixed being able to search alive teamleaders for intel
+   * :issue:`2556` Fixed broken mortars at airports
+   * :issue:`2561` Fixed fn_createAction using the incorrect hashmap key
+   * :issue:`2564` Fixed whiteout after alt-tab on maps using darkMapFix
+
+.. rubric:: Refactor
```
Comment (targetingsnake) [inline changelog.rst:241]: does this rubric belongs to any admunition?

**Review by targetingsnake — CHANGES_REQUESTED** [review]

File: changelog.rst [hunk 8]
```diff
@@ -8,6 +8,300 @@ Changelog
...
+
+.. note::
+   xxth Xxxxx 2022
+
+.. admonition:: Major
+
+.. rubric:: Conversion to Mod
+   * The mission has been converted to a full-blown mod to allow for access to new functionality and the ability for maps and templates to be created as addon mods instead of editing the mod itself. It also means that all officially integrated maps will be present with one mod, rather than across many separate mission files.
+   * Antistasi related keys can now properly be set within the settings
+   * Added 3rd party supports
```
Comment (HakonRydland) [inline changelog.rst:22]: Added support for 3rd party mods to extend Antistasi

File: changelog.rst [hunk 9]
```diff
@@ -8,6 +8,300 @@ Changelog
...
+   * :issue:`2181` Civilians are now created in the same way as soldiers allowing for greater customisation/themeing. The configuration for which is found in the civ template files.
+   * :issue:`2214` Garage system got improved with QoL changes, sorting and adjustments for the fuel system
+   * :issue:`2217` capturing a flag can be cancelled and logging for capturing got improved
+   * :issue:`2249` Seaports and Airbases can now own radio towers and thus jam radios.
+   * :issue:`2280` You can now take 5, 10, or 25 items at a time from the vehicle arsenal.
+   * :issue:`2305` Rebel AI can now be equipped when arsenal unlocks are disabled.
+   * :issue:`2306` The non-member limit for items in the arsenal is now configurable by the commander.
+   * :issue:`2318` Vehicle locking system overhauled, vehicles start unlocked by default.
+   * :issue:`2329` Added additional spec-ops groups for current and future use.
+   * :issue:`2356` Added buy menu vehicle render
```
Comment (HakonRydland) [inline changelog.rst:108]: Part of the buy menu point mentioned previously

File: changelog.rst [hunk 10]
```diff
@@ -8,6 +8,300 @@ Changelog
...
+   * :issue:`2510` Improved attack selection dependency on rebel HQ position
+   * :issue:`2521` Implemented AFK timeout parameter & status bar indicator to prevent AFK commanders blocking the progress
+   * :issue:`2523` KAT Medical implementation got updated to most current Kat - Advanced Medical REWRITE
+   * :issue:`2531` Adds logged in admins as members
+   * :issue:`2532` Balance utility trucks in cases where the civ factions lack them
+   * :issue:`2535` Increased default garage cap to 20 base + 4 per warlevel
+   * :issue:`2563` Added parameter for initial HR
+
+.. admonition:: Template updates
+   * Basically every template was touched up or overhauled :issue:`2181`, :issue:`2276`, :issue:`2277`, :issue:`2278`, :issue:`2302`, :issue:`2316`, :issue:`2333`, :issue:`2467`, :issue:`2475`
```
Comment (HakonRydland) [inline changelog.rst:133]: Drop the basically

File: changelog.rst [hunk 11]
```diff
@@ -8,6 +8,300 @@ Changelog
...
+      - moved markers
+   * Virolahti
+      - towns updated, added fuel stations, added bank locations, updated folder structure within sqm, updated vehicle markers, fixed broken marker names, removed edit-terrain-object-modules
+   * Chernarus_autumn
+      - added fresh map port
+   * Khe Sanh
+      - added fresh map port
+
+.. admonition:: Groundwork
+   * :issue:`2047` Moved templates into a separate addon
```
Comment (HakonRydland) [inline changelog.rst:176]: Incorrect, templates are still in core

**Review by HakonRydland — CHANGES_REQUESTED** [review]

File: changelog.rst (hunk 1 above)
Comment (Bob-Murphy) [reply changelog.rst:12]: solved

File: changelog.rst (hunk 2 above)
Comment (Bob-Murphy) [reply changelog.rst:91]: solved

File: changelog.rst (hunk 3 above)
Comment (Bob-Murphy) [reply changelog.rst:127]: solved

File: changelog.rst (hunk 4 above)
Comment (Bob-Murphy) [reply changelog.rst:139]: solved

File: changelog.rst (hunk 5 above)
Comment (Bob-Murphy) [reply changelog.rst:172]: solved

File: changelog.rst (hunk 6 above)
Comment (Bob-Murphy) [reply changelog.rst:199]: solved

File: changelog.rst (hunk 7 above)
Comment (Bob-Murphy) [reply changelog.rst]: solved

File: changelog.rst (hunk 9 above)
Comment (Bob-Murphy) [reply changelog.rst]: solved

File: changelog.rst (hunk 10 above)
Comment (Bob-Murphy) [reply changelog.rst]: solved

File: changelog.rst (hunk 11 above)
Comment (Bob-Murphy) [reply changelog.rst]: solved

File: changelog.rst (hunk 8 above)
Comment (Bob-Murphy) [reply changelog.rst]: solved

**Review by targetingsnake — COMMENTED** [review]
i'll found another bug with the sphinx renderer, i already notified @Giddius on discord,
the rubrics beneath admonition 'major' are just got converted to a bunch of text instead of a list

**Comment by targetingsnake** [comment]
``> i'll found another bug with the sphinx renderer, i already notified @Giddius on discord, the rubrics beneath admonition 'major' are just got converted to a bunch of text instead of a list

i spoke with giddi, we can solve the issue when rubrics and it's content are definied as following example:

```
  .. rubric:: Refactor

   * :issue:`2182` Refactored initZones to move relevant hardcoded map information to the map relevant files
   * :issue:`2238` Refactored BattleMenue to prevent conflicts with base game UIs
```

will open change requests for it

File: changelog.rst [hunk 12]
```diff
@@ -1,5 +1,299 @@
...
+
+Version 3.0.0
+===============
+
+.. note::
+   xxth Xxxxx 2022
+
+.. admonition:: Major
+
+   .. rubric:: Conversion to Mod
```
Comment (targetingsnake) [inline changelog.rst:14]: updated rubric as described in comments

File: changelog.rst [hunk 13]
```diff
@@ -1,5 +1,299 @@
...
+      * The mission has been converted to a full-blown mod to allow for access to new functionality and the ability for maps and templates to be created as addon mods instead of editing the mod itself. It also means that all officially integrated maps will be present with one mod, rather than across many separate mission files.
+      * Antistasi related keys can now properly be set within the settings
+      * Added support for 3rd party mods to extend Antistasi
+      * 
+      * 
+
+   .. rubric:: Added Campaign StartUp UI and safe functionality :issue:`2488`
```
Comment (targetingsnake) [inline changelog.rst:21]: updated rubric as described in comments

File: changelog.rst [hunk 14]
```diff
@@ -1,5 +1,299 @@
...
+      * Allows to select factions for the campaign from all available templates based on mods loaded
+      * Allows to have multiple safegames at the same time
+      * Allows to set and change parameters for the campaign
+      * Allows to set the initial HQ position before starting the campaign
+      * Allows to save outside of the vars-file
+      * Shows warning when loading a mismatched or outdated mission
+      * 
+      * 
+
+   .. rubric:: Complete rework of the attack & support system
```
Comment (targetingsnake) [inline changelog.rst:31]: updated rubric as described in comments and keep list indents with seperation

File: changelog.rst [hunk 15]
```diff
@@ -1,5 +1,299 @@
...
+         - Ground vehicle travel times reduced and infantry offload reliability improved.
+         - CAS rewritten for reliability and fairness.
+      * Attack target selection rewritten:
+         - Enemies can now attack rebel targets outside mission distance at reduced probability.
+      * Enemy HQ knowledge is now persistent:
+         - Enemies may gain knowledge of HQ when supports are called nearby, or from traitor missions.
+         - Once enemies are aware of the HQ, an HQ attack may be launched instead of a normal major attack.
+         - Moving HQ more than 1km away will reset the HQ knowledge. 
+      
+   .. rubric:: Fuel economy overhaul
```
Comment (targetingsnake) [inline changelog.rst:57]: updated rubric as described in comments

File: changelog.rst [hunk 16]
```diff
@@ -1,5 +1,299 @@
...
+      * Vehicles now spawn with a random amount of fuel in the tank and fuel stations are now present and usable on all maps. Containers can be bought to transport additional fuel.
+      * 
+      * 
+      
+   .. rubric:: New Buy Vehicle UI - :issue:`2259`
```
Comment (targetingsnake) [inline changelog.rst:62]: updated rubric as described in comments

File: changelog.rst [hunk 17]
```diff
@@ -1,5 +1,299 @@
...
+      * A new UI for buying vehicles has been developed which includes far more information than the current one.
+      * 
+      * 
+      
+   .. rubric:: Added Guest Commander functionality :issue:`2428`
```
Comment (targetingsnake) [inline changelog.rst:67]: updated rubric as described in comments

File: changelog.rst [hunk 18]
```diff
@@ -1,5 +1,299 @@
...
+      * this allows servers to be functional when the member system is enabled and no member is on the servers
+      * 
+      * 
+      
+   .. rubric:: New mod compatibilities
```
Comment (targetingsnake) [inline changelog.rst:72]: updated rubric as described in comments

File: changelog.rst [hunk 19]
```diff
@@ -1,5 +1,299 @@
...
+      * Added CUP templates - includes many factions like ACR, AFRF, BAF, CDF, ION, RACS, SLA, TKA, and US Army and US Marines.
+      * Added Global Mobilisation templates - includes the factions ...
+      * Added Unsung templates - includes the factions ...
+      
+   .. rubric:: New maps ports
```
Comment (targetingsnake) [inline changelog.rst:77]: updated rubric as described in comments

File: changelog.rst [hunk 20]
```diff
@@ -1,5 +1,299 @@
...
+      * Khe Sanh
+      * Chernarus Autumn
+      
+   .. rubric:: Translated Antistasi additional languages
```
Comment (targetingsnake) [inline changelog.rst:81]: updated rubric as described in comments

**Review by targetingsnake — CHANGES_REQUESTED** [review]

File: changelog.rst [hunk 21]
```diff
@@ -1,5 +1,299 @@
...
+      - Enabled buying loot crates from any rebel flag
+      - Fixed incorrect bounding box calc for load/unload
+      - Changed load/unload speed to be independent of script load
+   * :issue:`2453` Adds additional visible information during vehicle/asset placement
+   * :issue:`2454` Maru was removed. Petros is now called Petros on all maps, including Tanoa.
+   * :issue:`2469` Skip time now checks for active enemy AI instead of any enemy AI
+   * :issue:`2472` Tweaked ability of rebels to respond to punishments and major attacks
+   * :issue:`2477` Added finite rebel launchers and explosives
+   * :issue:`2503` Added parameters for initial player and rebel faction  money
+   * :issue:`2510` Improved attack selection dependency on rebel HQ position
```
Comment (jaj22) [inline changelog.rst:119]: Can go, tweak to new behaviour.

File: changelog.rst [hunk 22]
```diff
@@ -1,5 +1,299 @@
...
+   * :issue:`2181` Civilians are now created in the same way as soldiers allowing for greater customisation/themeing. The configuration for which is found in the civ template files.
+   * :issue:`2214` Garage system got improved with QoL changes, sorting and adjustments for the fuel system
+   * :issue:`2217` capturing a flag can be cancelled and logging for capturing got improved
+   * :issue:`2249` Seaports and Airbases can now own radio towers and thus jam radios.
+   * :issue:`2280` You can now take 5, 10, or 25 items at a time from the vehicle arsenal.
+   * :issue:`2305` Rebel AI can now be equipped when arsenal unlocks are disabled.
+   * :issue:`2306` The non-member limit for items in the arsenal is now configurable by the commander.
+   * :issue:`2318` Vehicle locking system overhauled, vehicles start unlocked by default.
+   * :issue:`2329` Added additional spec-ops groups for current and future use.
+   * :issue:`2381` Added parameters for enemyNearCheck
```
Comment (jaj22) [inline changelog.rst:104]: This one is a more substantial change than described, especially from the player perspective. enemyNearCheck now only considers enemies in combat mode, and (nearly?) every enemy proximity check now uses the same rules.

File: changelog.rst [hunk 23]
```diff
@@ -1,5 +1,299 @@
...
+      * Italian
+      * Korean
+      * Polish
+      * Russian
+      * Simplified Chinese
+      * Spanish
+
+.. admonition:: Minor
+
+   * :issue:`2114` implemented system that gets compatible magazines for a weapon
```
Comment (jaj22) [inline changelog.rst:93]: Maybe too internal. The related external changes were elsewhere: Arsenal now listing all compatible magazines for weapons (doesn't even use 2114 anymore, as Arma 2.10 added commands), and vehicle arsenal now listing all compatible magazines for vehicles (important for 3CB BAF).

File: changelog.rst [hunk 24]
```diff
@@ -1,5 +1,299 @@
...
+   * :issue:`2395` Added facewear support for AI loadouts
+   * :issue:`2418` Implemented QoL looting & logistic tweaks
+   * Switched loot crate carrying from forceWalk true to allowSprint false (about 2x faster movement)
+      - Enabled buying loot crates from any rebel flag
+      - Fixed incorrect bounding box calc for load/unload
+      - Changed load/unload speed to be independent of script load
+   * :issue:`2453` Adds additional visible information during vehicle/asset placement
+   * :issue:`2454` Maru was removed. Petros is now called Petros on all maps, including Tanoa.
+   * :issue:`2469` Skip time now checks for active enemy AI instead of any enemy AI
+   * :issue:`2472` Tweaked ability of rebels to respond to punishments and major attacks
```
Comment (jaj22) [inline changelog.rst:116]: Tweak to new behaviour, remove.

File: changelog.rst [hunk 25]
```diff
@@ -1,5 +1,299 @@
...
+   * :issue:`2047` Switched over to new template system
+   * :issue:`2153` Added functionality that formats a scalar as the specified length hexidecimal string
+   * :issue:`2168` Added a shortID generator
+   * :issue:`2174` & :issue:`2245` Improvements and fixes for the StreetArtist tool
+   * :issue:`2186` Added Garbage Collection Component of KeyCache.
+   * :issue:`2206` Removed the legacy KeyCache files
+   * :issue:`2229` Added feature toggle and assets for UI rework
+   * :issue:`2230` Added a build tool for the antistasi mod.
+   * :issue:`2270` Moved garage initServer to postInit
+   * :issue:`2297` Set Temperate as default for template auto selection
```
Comment (jaj22) [inline changelog.rst:183]: Superseded, dead.

File: changelog.rst [hunk 26]
```diff
@@ -1,5 +1,299 @@
...
+   * :issue:`2470` Fixed being able to kill players while carrying items
+   * :issue:`2472` Fixed JNA not using the compatibleMagazines command
+   * :issue:`2480` Fixed players getting stuck on large objects when carrying something
+   * :issue:`2490` Fixed enemy militia trucks not being sellable
+   * :issue:`2498` Fixed players being able to mount a static that is being carried
+   * :issue:`2508` Fixed an infinite money exploit
+   * :issue:`2517` Fixed being able to load game saves with missing mods
+   * :issue:`2537` Fixed Antistasi UI layer numbers fighting with other mods
+   * :issue:`2551` Fixed being able to search alive teamleaders for intel
+   * :issue:`2556` Fixed broken mortars at airports
```
Comment (jaj22) [inline changelog.rst:236]: Bug added since 2.5.5 so don't need to list.

File: changelog.rst [hunk 27]
```diff
@@ -1,5 +1,299 @@
...
+   * :issue:`2323` Fixed broken flag textures on AI resources and outposts
+   * :issue:`2326` Fixed players not being silent with ACE
+   * :issue:`2328` Fixed categoryOverrides not being created on clients
+   * :issue:`2340` Fixed multiple EHs being added in confirmPlacement
+   * :issue:`2344` Fixed incorrect variables used in flight height restrictions
+   * :issue:`2390` Fixed behaviour and remoteExec bugs in undercover AI
+   * :issue:`2392` Fixed bad JIP marker colours
+   * :issue:`2397` Fixed Petros face being used by other AI
+   * :issue:`2411` Fixed rebel static mounting on DS
+   * :issue:`2423` Fixed errors with event triggers
```
Comment (jaj22) [inline changelog.rst:222]: Pretty sure that was a post-2.5.5 bug.

File: changelog.rst [hunk 28]
```diff
@@ -1,5 +1,299 @@
...
+   * :issue:`2434` Fixed issues high command fast travel and garrison functionality
+   * :issue:`2436` Fixed arsenal weapon switch duping magazines
+   * :issue:`2449` Fixed issue with ACE cargo unloading of loot crates
+   * :issue:`2470` Fixed being able to kill players while carrying items
+   * :issue:`2472` Fixed JNA not using the compatibleMagazines command
+   * :issue:`2480` Fixed players getting stuck on large objects when carrying something
+   * :issue:`2490` Fixed enemy militia trucks not being sellable
+   * :issue:`2498` Fixed players being able to mount a static that is being carried
+   * :issue:`2508` Fixed an infinite money exploit
+   * :issue:`2517` Fixed being able to load game saves with missing mods
```
Comment (jaj22) [inline changelog.rst:233]: Very-Post-2.5.5 bug.

File: changelog.rst [hunk 29]
```diff
@@ -1,5 +1,299 @@
...
+   * :issue:`2263` Fixed issues with compatible magazine/ammunition detection
+   * :issue:`2281` Fixed issues with RHS asset stacking in the arsenal
+   * :issue:`2284` Fixed players being able to carry objects into vehicles
+   * :issue:`2290` Fixed roadblock vehicles despawning after stealing 
+   * :issue:`2292` Fixed cargo trucks not being sellable 
+   * :issue:`2296` Fixed town names in tasks and city info
+   * :issue:`2307` Fixed issue of garage code breaking pylons
+   * :issue:`2321` Fixed money displays breaking
+   * :issue:`2323` Fixed broken flag textures on AI resources and outposts
+   * :issue:`2326` Fixed players not being silent with ACE
```
Comment (jaj22) [inline changelog.rst:214]: I think this was a response to the face/voice patch.

Comment (Lazejun) [reply changelog.rst:214]: yep

**Review by HakonRydland — APPROVED** [review]

File: changelog.rst (hunk 29 above)
Comment (Bob-Murphy) [reply changelog.rst]: solved

File: changelog.rst (hunk 28 above)
Comment (Bob-Murphy) [reply changelog.rst]: solved

File: changelog.rst (hunk 27 above)
Comment (Bob-Murphy) [reply changelog.rst]: solved

File: changelog.rst (hunk 26 above)
Comment (Bob-Murphy) [reply changelog.rst]: solved

File: changelog.rst (hunk 25 above)
Comment (Bob-Murphy) [reply changelog.rst]: solved

File: changelog.rst (hunk 24 above)
Comment (Bob-Murphy) [reply changelog.rst]: solved

File: changelog.rst (hunk 23 above)
Comment (Bob-Murphy) [reply changelog.rst]: solved

File: changelog.rst (hunk 22 above)
Comment (Bob-Murphy) [reply changelog.rst]: solved

File: changelog.rst (hunk 21 above)
Comment (Bob-Murphy) [reply changelog.rst]: solved

File: changelog.rst (hunk 20 above)
Comment (Bob-Murphy) [reply changelog.rst:72]: solved

File: changelog.rst (hunk 19 above)
Comment (Bob-Murphy) [reply changelog.rst:68]: solved

File: changelog.rst (hunk 18 above)
Comment (Bob-Murphy) [reply changelog.rst:63]: solved

File: changelog.rst (hunk 17 above)
Comment (Bob-Murphy) [reply changelog.rst:60]: solved

File: changelog.rst (hunk 16 above)
Comment (Bob-Murphy) [reply changelog.rst:57]: solved

File: changelog.rst (hunk 15 above)
Comment (Bob-Murphy) [reply changelog.rst:53]: solved

File: changelog.rst (hunk 14 above)
Comment (Bob-Murphy) [reply changelog.rst:27]: solved

File: changelog.rst (hunk 13 above)
Comment (Bob-Murphy) [reply changelog.rst:19]: solved

File: changelog.rst (hunk 12 above)
Comment (Bob-Murphy) [reply changelog.rst:14]: solved

File: changelog.rst [hunk 30]
```diff
@@ -1,5 +1,283 @@
...
+   * Spanish
+
+.. admonition:: Minor
+
+   * :issue:`2178` Added ACE food and drink to the arsenal. (Food only with parameter)
+   * :issue:`2181` Civilians are now created in the same way as soldiers allowing for greater customisation/themeing. The configuration for which is found in the civ template files.
+   * :issue:`2214` Garage system got improved with QoL changes, sorting and adjustments for the fuel system
+   * :issue:`2217` capturing a flag can be cancelled and logging for capturing got improved
+   * :issue:`2249` Seaports and Airbases can now own radio towers and thus jam radios
+   * :issue:`2280` You can now take 5, 10, or 25 items at a time from the vehicle arsenal
```
Comment (Lazejun) [inline changelog.rst:89]: #2280 Could mention the Keybinds shift was 5, CTRL was 10 both is 25.
#2305 Could Rephrase that, they can now equip Items that are not Unlocked as soon as there is a Sufficient Amount.

Also they try to use Optics that are logical for the Equipment, no more Thermal SMGs, Sniper Optic SMGs etc.

The more you have of an Item the more likely they are to Equip it.

File: changelog.rst [hunk 31]
```diff
@@ -1,5 +1,283 @@
...
+
+.. admonition:: Template updates
+
+   * Every template was touched up or overhauled :issue:`2181`, :issue:`2276`, :issue:`2277`, :issue:`2278`, :issue:`2302`, :issue:`2316`, :issue:`2333`, :issue:`2467`, :issue:`2475`     
+      * Removed unused loadout creation stuff as its all handled by EquipRebell
+      * Removed comments as they can be found in Example Templates
+      * Updated format of the Rebel Example Template
+      * Added If cases for DLC uniforms for Vanilla and RHS
+      * Added a check in initVarServer for an empty civ helicopter as it will error with VN rebels
+      * Fixed a syntax error in the MACV template
```
Comment (Lazejun) [inline changelog.rst:123]: * Fixed a syntax error in the MACV template

Thats another in between Commits fix not work mentioning.

#2276 has nothing Todo with Templates.
#2277 and #2278, #2302, #2333 are all in 2.5.5. already not worth to mention here.
#2475 is an Inbetween changes fix, not worth being mentioned.

File: changelog.rst [hunk 32]
```diff
@@ -1,5 +1,283 @@
...
+
+   * :issue:`2178` Added ACE food and drink to the arsenal. (Food only with parameter)
+   * :issue:`2181` Civilians are now created in the same way as soldiers allowing for greater customisation/themeing. The configuration for which is found in the civ template files.
+   * :issue:`2214` Garage system got improved with QoL changes, sorting and adjustments for the fuel system
+   * :issue:`2217` capturing a flag can be cancelled and logging for capturing got improved
+   * :issue:`2249` Seaports and Airbases can now own radio towers and thus jam radios
+   * :issue:`2280` You can now take 5, 10, or 25 items at a time from the vehicle arsenal
+   * :issue:`2305` Rebel AI can now be equipped when arsenal unlocks are disabled
+   * :issue:`2306` The non-member limit for items in the arsenal is now configurable by the commander
+   * :issue:`2318` Vehicle locking system overhauled, vehicles start unlocked by default
```
Comment (Lazejun) [inline changelog.rst:92]: #2318 as Players love taking stuff out of Context this has to be more specific.

This is about Player Vehicles being locked by Default for other Players that are not part of your Group,
Which was changed to being Unlocked by Default for everyone, Players in your Group Ignore locks and Members ignore locks.

----------> They dont get kicked out of the Vehicle when they enter.

In general its more of a Feature for bigger Public servers, which is funny as probably most people dont know about it.

File: changelog.rst [hunk 33]
```diff
@@ -1,5 +1,283 @@
...
+   * Vehicles now spawn with a random amount of fuel in the tank and fuel stations are now present and usable on all maps. Containers can be bought to transport additional fuel.
+   * Fuel Stations also contain a limited amount of fuel to encourage players to use fuel wisely.
+      
+   .. rubric:: New Buy Vehicle UI - :issue:`2259`
+   * A new UI for buying vehicles has been developed which includes far more information than the current one.
+      
+   .. rubric:: Added Guest Commander functionality :issue:`2428`
+   * this allows servers to be functional when the member system is enabled and no member is on the servers
+      
+   .. rubric:: New mod compatibilities
```
Comment (Lazejun) [inline changelog.rst:63]: Mostly Pointless as you cant link to PRs Issues etc.
#2239 CUP
#2427 GM
#2379 Unsung

**Review by Lazejun — CHANGES_REQUESTED** [review]

File: changelog.rst (hunk 12 above)
Comment (targetingsnake) [reply changelog.rst:14]: no, two emoty lines benath the `.. rubric::` are still missing and are very important

File: changelog.rst (hunk 13 above)
Comment (targetingsnake) [reply changelog.rst:19]: no, two emoty lines benath the `.. rubric::` are still missing and are very important

File: changelog.rst (hunk 14 above)
Comment (targetingsnake) [reply changelog.rst:27]: no, two emoty lines benath the `.. rubric::` are still missing and are very important

File: changelog.rst (hunk 15 above)
Comment (targetingsnake) [reply changelog.rst:53]: no, two emoty lines benath the `.. rubric::` are still missing and are very important

File: changelog.rst (hunk 16 above)
Comment (targetingsnake) [reply changelog.rst:57]: no, two emoty lines benath the `.. rubric::` are still missing and are very important

File: changelog.rst (hunk 17 above)
Comment (targetingsnake) [reply changelog.rst:60]: no, two emoty lines benath the `.. rubric::` are still missing and are very important

File: changelog.rst (hunk 18 above)
Comment (targetingsnake) [reply changelog.rst:63]: no, two emoty lines benath the `.. rubric::` are still missing and are very important

File: changelog.rst (hunk 33 above)
Comment (targetingsnake) [reply changelog.rst:63]: afaik you can link to issues with the `:issue:` and then the number in specific high commata

File: changelog.rst (hunk 19 above)
Comment (targetingsnake) [reply changelog.rst:68]: no, two emoty lines benath the `.. rubric::` are still missing and are very important

File: changelog.rst (hunk 20 above)
Comment (targetingsnake) [reply changelog.rst:72]: no, two emoty lines benath the `.. rubric::` are still missing and are very important

**Review by targetingsnake — CHANGES_REQUESTED** [review]

**Review by killerswin2 — APPROVED** [review]

File: changelog.rst (hunk 12 above)
Comment (targetingsnake) [reply changelog.rst:14]: solved

File: changelog.rst (hunk 13 above)
Comment (targetingsnake) [reply changelog.rst:21]: solved

File: changelog.rst (hunk 14 above)
Comment (targetingsnake) [reply changelog.rst:31]: solved

File: changelog.rst (hunk 15 above)
Comment (targetingsnake) [reply changelog.rst:59]: solved

File: changelog.rst (hunk 16 above)
Comment (targetingsnake) [reply changelog.rst:65]: solved

File: changelog.rst (hunk 17 above)
Comment (targetingsnake) [reply changelog.rst:70]: solved

File: changelog.rst (hunk 18 above)
Comment (targetingsnake) [reply changelog.rst:75]: solved

File: changelog.rst (hunk 19 above)
Comment (targetingsnake) [reply changelog.rst:82]: solved

File: changelog.rst (hunk 20 above)
Comment (targetingsnake) [reply changelog.rst:88]: solved

**Review by targetingsnake — APPROVED** [review]

File: changelog.rst (hunk 30 above)
Comment (Bob-Murphy) [reply changelog.rst:107]: Do we have a parameter for 2305 / what is the "suffucient amount"?

File: changelog.rst (hunk 33 above)
Comment (Bob-Murphy) [reply changelog.rst:75]: solved

File: changelog.rst (hunk 32 above)
Comment (Bob-Murphy) [reply changelog.rst]: solved

File: changelog.rst (hunk 31 above)
Comment (Bob-Murphy) [reply changelog.rst]: solved

File: changelog.rst (hunk 30 above)
Comment (Lazejun) [reply changelog.rst]: No Parameter,
ITEM_MIN 10
ITEM_MAX 50

They start with 10, highest chances are at 50

File: changelog.rst (hunk 31 above)
Comment (Lazejun) [reply changelog.rst]: Should be good here

File: changelog.rst (hunk 32 above)
Comment (Lazejun) [reply changelog.rst]: ✔️

File: changelog.rst (hunk 33 above)
Comment (Lazejun) [reply changelog.rst:75]: ✔️

**Review by Lazejun — APPROVED** [review]

**Pull request merged**